common/src/test/java/org/opensearch/ml/common/transport/connector/MLCreateConnectorInputTests.java

```diff
@@ -24,19 +19,33 @@
 import org.opensearch.common.io.stream.BytesStreamOutput;
 import org.opensearch.common.settings.Settings;
 import org.opensearch.common.xcontent.LoggingDeprecationHandler;
-import org.opensearch.common.xcontent.XContentFactory;
 import org.opensearch.common.xcontent.XContentType;
 import org.opensearch.core.common.io.stream.StreamInput;
+import org.opensearch.core.xcontent.MediaTypeRegistry;
```

Comment:
I think we are keeping `XContentFactory`: https://github.com/opensearch-project/ml-commons/blob/2.x/common/src/test/java/org/opensearch/ml/common/transport/connector/MLCreateConnectorInputTests.java

Reply:
Done